auth-agents-common/src/main/java/org/apache/atlas/policytransformer/CachePolicyTransformerImpl.java

```diff
@@ -177,6 +180,16 @@ public ServicePolicies getPolicies(String serviceName, String pluginId, Long las
                     }
                 }
 
+                AtlasPerfMetrics.MetricRecorder recorder1 = RequestContext.get().startMetricRecord("filterPolicies");
+                //filter out policies based on serviceName
+                List<RangerPolicy> policiesA = allPolicies.stream().filter(x -> serviceName.equals(x.getService())).collect(Collectors.toList());
+                List<RangerPolicy> policiesB = allPolicies.stream().filter(x -> tagServiceName.equals(x.getService())).collect(Collectors.toList());
+
+                servicePolicies.setPolicies(policiesA);
+                servicePolicies.getTagPolicies().setPolicies(policiesB);
```

Comment:
Why not directly use "setTagPolicies"?

Reply:
TagPolicies is nested model which itself has policies field
<img width="763" alt="image" src="https://github.com/atlanhq/atlas-metastore/assets/89625355/13c97d1d-f3ad-430a-b76a-e412048ff77f">